charts/k8ssandra/templates/traefik.ingressroutes.yaml

```diff
@@ -40,6 +40,77 @@ spec:
   routes:
     - match: HostSNI(`*`)
       services:
-        - name: {{ $clusterName }}-{{ $datacenterName }}-service
+        - name: {{ $releaseName }}-{{ $datacenterName }}-service
+          port: 9042
+{{- end }}
+{{- if $traefik.stargate.enabled }}
+---
+{{- if and $traefik.stargate.cassandra.enabled $traefik.cassandra.enabled }}
+  {{- fail "ingress.traefik.stargate.cassandra.enabled and ingress.traefik.cassandra.enabled cannot both be enabled." }}
+{{- end }}
+{{- $stargateHost := required $traefik.stargate.host }}
```

Comment:
The `required` function expects two arguments. This will cause a templating error.

Reply:
Yep, found and fixed. Thanks. :)